src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -718,4 +718,35 @@ public boolean hasIndicatedSections(String courseId) throws EntityDoesNotExistEx
         }
         return archivedCourseIds;
     }
+
+    /**
+     * Check that {@code courseTimeZone} is valid and then return a {@link CourseAttributes}
```

Comment:
Check**s**, return**s**. End the sentence with a full stop.

Reply:
My apology for not following the javadoc standard. Will take note next time.